orc8r/cloud/Makefile

```diff
@@ -175,7 +175,7 @@ else ifeq ($(OS_NAME),Darwin)
 endif
 
 nms_prereqs_ubuntu:
-	curl -sL https://deb.nodesource.com/setup_17.x | bash -
+	curl -sL https://deb.nodesource.com/setup_18.x | bash -
```

Comment:
It seems we can't use node 18 if we upgrade to bionic:
https://github.com/nodesource/distributions/issues/1392